src/os/feature/feature.js

```diff
@@ -528,19 +528,28 @@ const createLineOfBearing = function(feature, opt_replace, opt_lobOpts) {
 const createRings = function(feature, opt_replace) {
   var ringGeom;
 
-  if (!opt_replace) {
-    ringGeom = /** @type {GeometryCollection} */ (feature.values_[RecordField.RING]);
-    if (ringGeom instanceof GeometryCollection) {
-      return ringGeom;
+  if (feature) {
+    if (!opt_replace) {
+      ringGeom = /** @type {GeometryCollection} */ (feature.values_[RecordField.RING]);
+      if (ringGeom instanceof GeometryCollection) {
+        return ringGeom;
+      }
     }
-  }
 
-  if (feature) {
     cleanRingGeoms(feature);
 
     var options = /** @type {osx.feature.RingOptions} */ (feature.get(RecordField.RING_OPTIONS));
     var geometry = feature.getGeometry();
-    var center = geometry ? toLonLat(olExtent.getCenter(geometry.getExtent()), osMap.PROJECTION) : null;
+    var center = null;
+
+    if (geometry && geometry instanceof Point) {
+      center = toLonLat(olExtent.getCenter(geometry.getExtent()), osMap.PROJECTION);
```

Comment:
For a Point feature, `geometry.getCoordinates()` should be the same as getting the extent center.